app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,38 @@
+require 'net/http'
+require 'json'
+require 'uri'
+
+class AuthorsController < ActionController::Base
```

Comment:
Please add controller and request tests.

Reply:
There is still no test coverage for the functionality of your controller. I can remove the main line and your tests won't catch it.